### DISCUSSION TITLE
搞成这样，还是大家这么多年被高房价，高房租 折磨的太狠了

### DISCUSSION THREAD

**Discussion by SevenChen** [WeNeedHome/SummaryOfLoanSuspension#396] [General]
压的越久，炸的越狠

**Comment by msidolphin** [comment]
今年差点上车，订房前一天晚上后悔了，感觉要把一辈子都搭上去😭

**Reply by Jackwew** [reply]
上个月刚上车，怎么办

**Reply by libaicn** [reply]
挺住

**Reply by DDFighting** [reply]
这玩意没法说，大头还是进地方政府腰包了

**Reply by lazyee** [reply]
> 今年差点上车，订房前一天晚上后悔了，感觉要把一辈子都搭上去😭

2年前我在老家买房子的时候,就很怕烂尾,最后想了很久还是上车了二手房

**Comment by renjianbingqi** [comment]
主要还是战狼这两年，搞的白左这么好骗的群体，都把订单转去 东南亚七国加印度了， 日韩台也都转过去了。
